### ISSUE TITLE
Temperature limit setting for extruder and bed

### ISSUE THREAD

**Issue by HappyMark93** [SoftFever/OrcaSlicer#2617]
I noticed that there is no option to set max. allowable temperature of extruder and heat bed. 

**Which printers will be beneficial to this feature**
This option will be beneficial to all who have extruder with ptfe tube insert, especially for beginner users if it come preset with printer profiles, and for generic marlin and voron should  be set to 230 C for extruder.

**Describe the solution you'd like**
Option to set max allowable temperature for heat bed and extruder.
Warning when trying to change this temperature with description of potential issues.
Warning when temperature for filament is set above this temperature.
Exporting g-code with temperature setting exceeding this temperature not passible even if it is from user script(beginner user might copied something from web without understanding exactly the meaning .

This protection when set correctly is available in firmware but only at execution time, it might be too late to see this mistake in print? 
Or even worse it is not set in firmware at all.

And I am writing this because default setting for generic pet-g is 255 C which is too much for my bowden printer, I print it at 230 with no problems.

Some will tell that 260 C is limit for PTFE, and I agree that it is limit for Mechanical properties of good quality PTFE, as well that toxic fume evaporation at that temperature is relatively low, but steel it is up to 20 times more fumes per hour then on 230 C, in comparing with 300 C where there is 50 times more fumes per hour then on 230 C and it is considered that when working with PTFE on 300 C a good ventilation is a must.

Now imagine how bad can it be when printing 10 hours in non vented room on cold winter day and can not open window because print will fail immediately.

**Comment by HappyMark93** [comment]
I absolutely agree, but updating printer firmware settings just because extruder swap is un necessary. Slicer limit is simple, and with default value set to safe limit (for generic 3D printer) or factory recommended it is great addition to safety and functionality of slicer. It can give new user warning in advance instead of realizing that temp limit is reached only after print is started.  

This protection when set correctly is available in firmware but only at execution time, it might be too late to see this mistake in print? (Printer is not in same room or building. Printer is used by multiple users at hub....)
Or even worse it is not set in firmware at all.

Basically it gives ability that one can completely configure extruder as second printer with all parameters needed.
